cfme/fixtures/embedded_ansible.py

```diff
@@ -0,0 +1,81 @@
+import pytest
+from wrapanapi.utils.random import random_name
+
+
+def open_namespace(domain_name, namespace_levels):
+    namespace = domain_name
+    for namespace_name in namespace_levels:
+        namespace = namespace.namespaces.instantiate(namespace_name)
+    return namespace
+
+
+def copy_ae_instance(appliance, instance_fqdn, domain_name):
+    """
+    Copy AE Instance into a domain.
+
+    :param appliance: appliance
+    :param instance_fqdn: (string) full path to the exisitng AE Instance that we want to copy e.g.
+                                   'ManageIQ/System/Event/EmsEvent/Nuage/nuage_enterprise_create'
+    :param domain_name: name of exisitng AE Domain where we want to copy to e.g. 'TestDomain'
+    :return: the new AE Instance
+    """
+    source = instance_fqdn.split('/')
+    source_domain_name = source.pop(0)
+    source_instance_name = source.pop(-1)
+    source_class_name = source.pop(-1)
+    source_domain = appliance.collections.domains.instantiate(source_domain_name)
+    namespace = open_namespace(source_domain, source)
+    namespace.classes.instantiate(source_class_name)\
```

Comment:
Please do not use `\` line continuation, wrap with parenthesis or change syntax of calls.

Also this call chain is not clear - please include a comment in-line in plain english of what resource you're trying to use and copy